modular_skyrat/modules/reagent_forging/code/anvil.dm

```diff
@@ -15,50 +16,46 @@
 		icon_state = "primitive_anvil_empty"
 
 /obj/structure/reagent_anvil/attackby(obj/item/I, mob/living/user, params)
-	var/obj/item/forging/incomplete/searchIncompleteSrc = locate(/obj/item/forging/incomplete) in contents
-	if(istype(I, /obj/item/forging/hammer) && searchIncompleteSrc)
+	var/obj/item/forging/incomplete/search_incomplete_src = locate(/obj/item/forging/incomplete) in contents
+	if(istype(I, /obj/item/forging/hammer) && search_incomplete_src)
 		playsound(src, 'modular_skyrat/modules/reagent_forging/sound/forge.ogg', 50, TRUE)
-		if(searchIncompleteSrc.heat_world_compare <= world.time)
+		if(COOLDOWN_FINISHED(search_incomplete_src, heating_remainder))
 			to_chat(user, span_warning("You mess up, the metal was too cool!"))
```

Comment:
Would this be better as a balloon alert? Usually warnings have been leaning more towards visuals like that rather than stuff that can be buried in chat.
something like balloon_alert("metal too cold!") or something simple?